pkg/cloud/volumeWaitParameters.go

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
super np: file name should be `volume_wait_parameters` to keep it consistent with other files in repo.

Reply:
Thanks for the catch, my brain switched languages for a second.